File: Server/MVC/Models/UserCred.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace MVC.Models;

[NotMapped]
public class UserCred
{
    public string Username { get; set; }

    public string Password { get; set; }

    public string? Nickname { get; set; }
    
    public string? Server { get; set; }
}

File: Android/app/src/main/java/com/example/android/entities/UserCred.java
package com.example.android.entities;

public class UserCred
{
    public String username;
    public String password;
    public String nickname;
    public String server;
    public String firebaseToken;

    public UserCred(String username, String password, String nickname, String server, String firebaseToken){
        this.username = username;
        this.password = password;
        this.nickname = nickname;
        this.server = server;
        this.firebaseToken = firebaseToken;
    }
}